### ISSUE TITLE
Add Option to hide heading if only one section is shown

### ISSUE THREAD

**Issue by Cadair** [alphapapa/org-super-agenda#64]
If only the "other items" header is being shown, the heading is a little superfluous in my agenda, it would be nice to be able to hide it.

**Comment by alphapapa** [comment]
Thanks, not a high priority for me now, but I may get to it sometime.

**Comment by pstray** [comment]
I agree with @Cadair here. Would be nice to get rid of the "Other items" heading when it is the only one.

**Comment by alphapapa** [comment]
> Would be nice if any single header could be removed too,

That is covered in the readme.

**Comment by pstray** [comment]
Is it? Please direct me to where. In which this case, this issue is a non-issue. "Any single header" as in removing the header of ANY group if it is the only group displayed is a wider problem than just removing the header for "Other items" if that group is the only one displayed (since it always is the last one).
